graylog2-web-interface/src/views/components/SearchBar.jsx

```diff
@@ -41,15 +34,14 @@ type Props = {
     },
   },
   disableSearch: boolean,
-  onExecute: (View) => void,
+  performSearch: () => void,
   queryFilters: Immutable.Map,
 };
 
-const SearchBar = ({ availableStreams, config, currentQuery, disableSearch = false, onExecute, queryFilters }: Props) => {
+const SearchBar = ({ availableStreams, config, currentQuery, disableSearch = false, performSearch, queryFilters }: Props) => {
```

Comment:
I think `onExecute` is more appropriate here, as it follows react conventions (`onEvent`, where the event is triggering an execution).

Reply:
Alright, I thought `performSearch` is more descriptive, but I get your point and renamed the prop.